# Testing collections.defaultdict and not with

In this code I was trying to pass an argument and simulate an error when trying to access a dictionary index with the argument, the result was an error as expected. But the result from defaultdict was not expected (because I didn't believe it was possible)